### PR TITLE
Update to v3.0.0-rc.16

### PR DESCRIPTION
**Pull request by fvasco**
Rc update to version 3.0.0-rc.16

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>ui-grid</artifactId>
-    <version>3.0.1-rc.11-SNAPSHOT</version>
+    <version>3.0.1-rc.16-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: This should be `3.0.0-rc.16-SNAPSHOT`

Comment (fvasco) [inline pom.xml]: Previous version is 3.0.0-rc.11, this should be 3.0.0-rc.16, are you agree?

Comment (jamesward) [inline pom.xml]: This should be `3.0.0-rc.16-SNAPSHOT` since Maven does the actual release (strips off the `-SNAPSHOT`).

**Comment by fvasco** [comment]
Version fixed to 3.0.0-rc.16-SNAPSHOT (commit amended)

**Comment by fvasco** [comment]
Thank you

**Pull request merged**